packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec

```diff
@@ -98,6 +98,9 @@ install -m 0644 .%{gem_instdir}/config/%{plugin_name}.yml \
 %{gem_instdir}/test
 
 %changelog
+* Thu Feb 13 2019 Zach Huntington-Meath - 0.21-0.3.pre.master
+- Unify prerelease macro handling
```

Comment:
this is probably not the changelog you were planning to add :)

Reply:
hah, it was not